internal/services/netapp/_maintainers/README.md

```diff
@@ -0,0 +1,53 @@
+# This is a maintainer file for the Azure NetApp Files service.
+
+This document gives insights into who is maintaining this service and includes details for why certain decisions in this service package have been made.
+
+## Mainteiners
+
+- paulomarquesc
+- orexii
+
+## Acceptance Tests
+
+- There is lack of SMB-related acceptance tests because it requires Active Directory Domain Controller infrastructure which is not easily automatable. SMB-related tests can only be tested if the infrastructure is setup beforehand which is not that trivial. We should not require SMB tests unless it comes with Domain Controller setup automation. Without automation, the SMB acceptance tests will fail and cause disruptions in CI/bulk testing.
+
+- New tests failing should not be accepted.
+
+## Polling functions
+
+- Some Netapp resources requires an extra type of polling mechanism. For example:
+
+  ```go
+    // Waiting for volume be completely provisioned
+    if err := waitForVolumeCreateOrUpdate(ctx, client, id); err != nil {
+        return err
+    }
+  ```
+
+  This is because some operations return from regular SDK polling as completed but due to several factors it is still in progress (e.g. ARM caching, software and hardware layer sync delays, etc.). These wait functions are necessary and should not be removed.
```

Comment:
The formatting here got weird (some words have colors) and I'm worried it'll cause issues when people view the file? I believe it's because of the above code block being tabbed over but you'll have to mess with it and see

Reply:
VS Code Markdown preview it looks ok:

![image](https://github.com/user-attachments/assets/1212fcae-7551-423d-b1da-4da225d0fb9c)

Reply:
I think it is some bug on GitHub PR interface because if you also navigate in my branch itself, it shows up normal:

![image](https://github.com/user-attachments/assets/c7a4977b-b26d-4ded-bb21-8e2278daa991)